stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -767,14 +937,41 @@ public final class _TensorComputation {
 }
 
 private extension _TensorComputation {
-  /// Runs the tensor program. Aborts the process on error, and emits an error
-  /// string to STDERR.
+  /// In eager mode, runs a device function in a corresponding thread given by
+  /// `threadIdx`. Otherwise, runs all devices functions of the tensor
+  /// program. Aborts the process on error, and emits an error string to STDERR.
   // NOTE: This is to be called by the initializer. The computation gets
   // executed on initialization, thus this method will not be exposed to users.
-  private func execute() {
+  private func execute(threadIdx: Int) {
+    debugLog("Executing thread \(threadIdx).")
+    if let stateTFE = stateTFE {
+      internalConsistencyCheck(_RuntimeConfig.usesTFEagerAPI)
+      internalConsistencyCheck(threadIdx <= helperFunctionCount)
+      let op = stateTFE.ops[threadIdx]
+      if threadIdx == 0 {
+        var returnValueCount = Int32(returnValues.count)
+        TFE_Execute(op, &returnValues, &returnValueCount, status)
+        debugLog("""
+                   returnValues.count=\(returnValues.count), \
```

Comment:
Instead of indenting all the way through `"""` like this, we've been using a different style for multi-line string literals:

```swift
debugLog("""
  returnValues.count=\(returnValues.count), \
  returnValueCount=\(returnValueCount).
  """)
```

Reply:
Changed, though I should note that my emacs auto indentation picked the former. So it'd be nice if we don't have to "fight the tool".

Reply:
Well, emacs doesn't really enforce the standard Swift format. I doubt emacs' swift-mode even has support for multi-line string literals in Swift, which were introduced not too long ago.

If the tool was `swift-format` or `swiftlint`, then I would definite agree that not fighting the tool is ideal. Hopefully they will be available soon.

Reply:
SG